src/utils.ts

```diff
@@ -64,13 +66,43 @@ export const buildTransaction = (
     if (flags.multiPassphrases) {
         const passphrases: string[] = flags.multiPassphrases.split(";");
 
+        builder.senderPublicKey(
+            Identities.PublicKey.fromMultiSignatureAsset({
+                min: flags.min || 2,
+                publicKeys: passphrases.map((passphrase: string) => Identities.PublicKey.fromPassphrase(passphrase)),
+            }),
+        );
+
         for (let i = 0; i < passphrases.length; i++) {
             builder.multiSign(passphrases[i], i);
         }
+
+        builder.sign(passphrases[0]);
+
+        // todo
+        // if (type === "multiSignature") {
+        //     builder.sign(passphrases[0]);
+        // }
     }
 
     const transaction: Interfaces.ITransaction = builder.build();
 
+    if (!transaction.verify()) {
```

Comment:
We could do:
https://github.com/ArkEcosystem/core/blob/develop/packages/core-state/src/wallets/wallet.ts#L136

if it's a multisignature transaction including `verify()` if it's type 4.